src/snowcli/output/printing.py

```diff
@@ -23,6 +23,75 @@ def default(self, o):
         return super().default(o)
 
 
+class OutputData:
```

Comment:
Maybe we can reduce the output printing to `print(OutputData(...))`? This can be done by implementation of `__str__`. In this way we can hide the implementation from users.

Reply:
I like the idea, but it is not that easy to do it. We are using print(table) and print_jason from rich :/

Reply:
I agree, but think it should be done in other PR